HIbernate/MailSenderTask2/src/main/java/com/demo/model/MailSender.java

```diff
@@ -0,0 +1,138 @@
+package com.demo.model;
+
+
+import java.util.Date;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.persistence.Column;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+
+import com.fasterxml.jackson.annotation.JsonFormat; 
+
+@Entity
+@Table(name = "mailsender")
+public class MailSender {
```

Comment:
Not correct name for entity and table.
I think this is what you were trying to do:

@ Table(name="sent_mail")
public class SentMail {
}